projects/plugins/protect/src/js/components/interstitial/index.jsx

```diff
@@ -58,10 +59,19 @@ const SecurityBundle = ( { onAdd, redirecting, rest } ) => {
  * @returns {React.Component} Interstitial react component.
  */
 const Interstitial = ( { onSecurityAdd, securityJustAdded } ) => {
+	const [ isLowerThanLarge ] = useBreakpointMatch( 'lg', '<=' );
```

Comment:
This is right? You're using `<=`, which means everything, and not lower than large.

Reply:
it's WIP PR.

Reply:
I've removed the second arg in the hook.